quiz.rb

```diff
@@ -2,19 +2,66 @@
 # Create a class called Dog. It should have, as attributes, name, breed, and age.
 # Make all of those attributes read-only.
 
+class Dog
+  def initialize(name, breed, age)
+    @name = name
+    @breed = breed
+    @age = age
+  end
+
+  def name
+    @name
+  end
+
+  def age
+    @age
+  end
+
+  def breed
```

Comment:
This isn't the wrong answer - it does meet the spec - but it probably would have been simpler to just use `attr_reader` here.